packages/ui/package.json

```diff
@@ -21,7 +21,7 @@
     "@types/react-router-dom": "^5.0.1",
     "bootstrap": "^4.3.1",
     "bootstrap-components": "^1.1.287",
-    "casper-client-sdk": "^1.0.9",
+    "casper-client-sdk": "^1.0.10",
```

Comment:
Should this be updated to ^1.0.11?

Reply:
I have changed it to use local sdk from the repo.